packages/homepage/src/pages/_tweets.js

```diff
@@ -0,0 +1,232 @@
+const tweets = [
+  {
+    name: 'Addy Osmani',
+    quote: 'CodeSandbox is a constant joy to use.',
+    URL: 'https://twitter.com/addyosmani/status/1164360161114218496?s=20',
```

Comment:
Maybe just save the id instead of the full url, since we're also saving the username and we could recreate the link ourselves? 🤔